examples/sweeps/sweep_test.go

```diff
@@ -0,0 +1,96 @@
+package sweeps
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/moovfinancial/moov-go/pkg/moov"
+)
+
+func Example_SweepConfigs_Endpoints() {
+	mc, err := moov.NewClient()
+	if err != nil {
+		fmt.Printf("new client: %v", err)
```

Comment:
t.Error / t.Fatal ? We can't proceed without a client.

Reply:
Go's Example funcs can be used as tests, by [specifying output](https://github.com/moov-io/ach/blob/master/examples/example_ackRead_test.go#L54) that's expected. 

We've used Tests elsewhere in examples to leverage `*testing.T`.

Reply:
Yeah, but in this example, all of the IDs are just randomly generated UUIDs so this func is never meant to be run -- moreso an example users can use as a reference as they're building their integration.